inference-engine/tests/functional/inference_engine/ngraph_reshape_tests.cpp

```diff
@@ -471,7 +471,7 @@ TEST_F(NGraphReshapeTests, TestInterpParameters) {
     cnn.reshape(inShape);
 }
 
-TEST_F(NGraphReshapeTests, ReshapeWithDefaultGenericOps) {
+TEST_F(NGraphReshapeTests, DISABLED_ReshapeWithDefaultGenericOps) {
```

Comment:
Should we remove this test completely? If no then why it is disabled?

Reply:
updated